packages/circuits/package.json

```diff
@@ -19,6 +19,7 @@
     ],
     "scripts": {
         "pre": "node scripts/prepare",
+        "buildTest": "TEST=1 yarn build",
```

Comment:
I think it can be executed before `test`?
So the test script become `yarn buildTest && mocha -r ts-node/register test/*.test.ts --exit`

Reply:
Ah, i prefer not. I often test using `yarn test --grep 'test name'`, so i might not want to build all circuits.

Reply:
but it would fail if developers clone can run `yarn build` and `yarn test`
we should either skip these tests locally without building circuits or running these tests and building circuits.

Reply:
Sure, maybe build all circuits by default and remove the test only circuits as a part of the `pre` script

Reply:
[c7bf309](https://github.com/Unirep/Unirep/pull/313/commits/c7bf3099bb1bd7229622f7353c1223bfd61d0f54)